cmd/instance_create.go

```diff
@@ -73,12 +52,12 @@ If you wish to use a custom format, the available fields are:
 			os.Exit(1)
 		}
 
-		if hostnameCreate != "" {
-			if utility.ValidNameLength(hostnameCreate) {
+		if len(args) > 0 {
```

Comment:
I think we can change this to use https://github.com/civo/cli/blob/fa542ad26368fe8fe797f4057912a1e655525b82/utility/random.go#L44 in the case of the user don't pass the hostname

Reply:
@alejandrojnm if the user doesn't pass the hostname it will be automatically generated from - https://github.com/civo/civogo/blob/master/instance.go#L176 

it is tested and verified 👍